djangocms_versioning_filer/monkeypatch/admin/fileadmin.py

```diff
@@ -5,23 +5,6 @@
 from ...helpers import check_file_label_exists_in_folder
 
 
-def save_model(func):
```

Comment:
You can't delete this, it's not part of this PR!

Reply:
revertedthis change back